labs/aries-fabric.md

```diff
@@ -0,0 +1,29 @@
+# Aries Wrapper For Fabric
+
+# Description
+
+Self - Sovereign Identity application is an application by which you are able to keep your identity secured and at the same time also prove your identity to the verifier without revealing them. These applications are built using Hyperledger Aries, which depends on Indy as a Distributed Ledger and Ursa as a cryptography library. Currently, the most used Blockchain platform is Hyperledger Fabric. And the businesses that are building solutions using Fabric want to have some sort of identity verification to be done in their platform. This is the reason, why this project has come into existence. Once done it would enable Aries to interact with Fabric as a ledger, which would then enable SSI applications to be built on top of Fabric.
+
+# Scope of Lab
+
+As Hyperledger's mission is to increase the adoption of Blockchain technologies for business, this project would enable business that are using Fabric for building their blockchain solutions to have identity verification process to be in-built in their system. The future developements that it would undergo once it is set as a project in lab are :
+
+Currently AFJ only supports did, schema and credential definition as domain transactions. Once revocation and attrib transactions are added, we would add support for those as well in the chaincode and module.
+
+Now there are 2 separate wallets – indy wallet for agent and fabric wallet for client application, we would try to integrate that into one.
+
+We would also try to add support such that AFJ works with other ledger as well.
+
+# Initial Committers
+
+https://github.com/HarshMultani
+
+# Sponsor
+
+Kamlesh Nagware ( https://github.com/knagware9 ) - Role ( TSC Member )
+
+# Pre-existing repository
+
+https://github.com/HarshMultani/aries-fabric-wrapper
```

Comment:
This does not seem to be an existing repo. Results in 404.

Reply:
@tkuhrt  maybe it's a private repo, I can access this repo. @HarshMultani  you can make it public.

Reply:
I have made it public. Now, can you try to access it.

Reply:
Thanks, @HarshMultani 

One last thing, it looks like none of the commits in this repository contain DCO signoff. Hyperledger requires that all contributed code has a DCO sign off. Looking at these repositories, it appears that this is not the case. There are two options that we normally consider for fixing this:

1. Amend existing commits in the repository to include DCO sign off
2. Manually transfer the code from the existing repo to the newly created repo within the hyperledger-labs GitHub organization.

Reply:
@HarshMultani : Can you let us know which of these you would like to proceed with (1 or 2)? This is the last remaining things before we can move forward with this lab.

Reply:
Hi @tkuhrt, I would go with the first option, and ammend the commits with dco sign off in this week.

Reply:
Sounds good, @HarshMultani. Please let me know when that is done, and I will approve the lab.

Reply:
Hi @tkuhrt , I have created a new repo on the same link, which contains commits with DCO signoff. 
Can you check it?